config.yaml

```diff
@@ -120,6 +120,7 @@ sync:
 
 
 userscripts:
+    setup: bash ./copy_nmls.sh
```

Comment:
Hack new PFT lookup into place

Reply:
Why not point to the new file? Even if it is in your own directory for now.